python/tests/sklearn/preprocessing/data_test.py

```diff
@@ -486,60 +423,6 @@ def feature_extractor_test(self):
         self.assertEqual(expected_model['attributes']['input_shapes']['data_shape'][1]['isNullable'],
                      model['attributes']['input_shapes']['data_shape'][1]['isNullable'])
 
-    def imputer_test(self):
```

Comment:
This test was moved to `python/tests/sklearn/preprocessing/one_hot_encoder_test.py`